pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -666,6 +666,22 @@ func createBackupScheduleHook(
 	if !ok {
 		return nil, nil, nil, false, nil
 	}
+
+	// Replace any unqualified table names with partially qualified table names by adding the database
+	// name as a prefix
+	// E.g.: table name "mytbl" in database "mydb" -> "mydb.mytbl"
+	if schedule.Targets != nil {
+		for i, table := range schedule.Targets.Tables {
+			if len(strings.Split(table.String(), ".")) == 1 {
```

Comment:
Might want to get a SQL reviewer or @knz to chime in on how best to decide if we need to qualify it (or maybe we should just always run it through some canonicalization?)

Reply:
Yeah, @annezhu98 and I discussed this a little more so that we use the more canonical qualification methods. She is going to rework the PR with those changes.